scss/titles.scss

```diff
@@ -4,8 +4,9 @@ $text-big: 24px;
 
 .title {
   font-size: 24px;
-    padding: 32px 0 32px 50px;
-    position: relative;
+  padding: 32px 0 32px 50px;
+  position: relative;
+  margin: 50px 0 5px 50px;
```

Comment:
the 50 at top and left should be added by a padding in the body, I think you want more than 5px to separate the title from the techs